interface/elements/msc-simple-login-view/msc-simple-login-view.js

```diff
@@ -30,7 +30,7 @@ Polymer({
       .to('chainVersion')
       .to('syncStatus')
 
-    nwin.maximize();
+    //nwin.maximize();
```

Comment:
I think I did this in order to optimise for screens with better res. But it looks equally good now. Thanks!